web/app_options.js

```diff
@@ -230,6 +230,11 @@ const defaultOptions = {
     value: 1,
     kind: OptionKind.API,
   },
+  enableXfa: {
+    /** @type {boolean} */
+    value: false,
+    kind: OptionKind.API,
+  },
```

Comment:
Please take care to sort options *alphabetically* within each OptionKind section in this file.